libcodechecker/server/api/report_server.py

```diff
@@ -2093,6 +2102,19 @@ def __store_reports(self, session, report_dir, source_root, run_id,
         already_added = set()
         new_bug_hashes = set()
 
+        # Get checker names which was enabled during the analysis.
+        enabled_checkers = []
```

Comment:
Could we use a dict or set here instead of a list for `enabled_checkers` and `disabled_checkers`?